cluster/ci/config/prow/openshift/config_updater_rbac.yaml

```diff
@@ -128,7 +136,6 @@ items:
       resources:
         - serviceaccounts
         - services
-        - routes
```

Comment:
Note that this was "redundant" already, see https://github.com/openshift/release/pull/4482/files#diff-60a3d06fe4fab6062600ec22fd493d0cL105